### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by musangamfure**
Hello,

In this PR, I have worked on the styling feature to match the wireframe style: 
bookstore

Kindly, Check and let me know what you think.

Best Regards.

Happy Coding🎉

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -5,40 +5,37 @@ import PropTypes from 'prop-types';
...
+            <div className="progres__text">
+              <h2>65 %</h2>
               <p>Completed</p>
             </div>
           </div>
-          <div className="section3">
+          <div className="section__three">
             <p>CURRENT CHAPTER</p>
             <h3>Chapter 17</h3>
             <button type="button">Update Progress</button>
```
Comment (Felix45) [inline src/components/Books.js:41]: - [ Required ] Good job styling the application, however, I've noted some differences between your design and the design on Zeplin. Please see the list and the screenshot below for details on this.

  ## Navigation menu
  - [x] The background color of the navigation menu on the Zeplin design is `#ffffff` in color whereas the navigation menu on your design has a `#fafafa` background color.
  
  - [x] The Zeplin design has a user icon on the top right section of the navigation menu which has the shape of a `circle` whereas your design has a user icon that is `oval` in shape.
 
  - [x] The Zeplin design has a  grayish `border-bottom` on the navigation menu whereas your design has no clear border that distinguishes it from the body of the booklist.
  
  - [x] The text on the navigation menu `Books` and `Categories` should be left aligned, currently, the text on the menu is centered using the flex box property `space-between` please use `flex-start` to align them to the left.
  
  ## Add book form
  - [x] The alignment of the components in the Zeplin design has the same margin or padding that makes them appear on the same line both on the `left` and the `right`. The `add book` form's submit button should be right aligned and be on the same line as the right edge of the `BookList` and `User icon`
  
  ## Book list
  - [x] The following text `Book title`, `Book author` and `Comments` should be left aligned and aligned on the same line. Currently, the `Comment` text has a `padding` of `20px` which makes it not to be on the same line as the `Book title` and `Book author` text.
  

## Zeplin design 
Screenshot from 2023-06-17 17-35-15


## Your design
Screenshot from 2023-06-17 17-22-25

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @musangamfure, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [x] Good UI design :white_check_mark: 
- [x] Git flow is used correctly :white_check_mark: 
- [x] No linter errors :white_check_mark:  


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 2]
```diff
@@ -31,138 +34,240 @@
...
+  border-radius: 100%;
+  display: flex;
+  margin-left: 300px;
+}
+
+.nav__menu {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
```
Comment (brainconnect93) [inline src/App.css]: - [ ] Great effort working on the styling of the bookstore and some of the changes requested by the previous reviewer. But according to the previous reviewer, the user icon should be on the right-hand side and align with the books on a straight line vertically
- [ ] Also the `Add book` button in the add book form should be aligned on a straight line with the books added vertically. With this, you will be sure you replicate the design as seen on Zeplin perfectly.


| Your Design 👇| Zeplin Design 👇|
| ------------- | ------------- |
| book-emma  | zep-org  |

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @musangamfure, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️
- Styling of the bookstore CMS project ✔️
- Zeplin Design template used for the design ✔️
- There is a good use of Git flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @musangamfure ,

Great job so far! 

May I highlight the following: 
- [x] Your Bookstore looks awesome 👏  ✔️ 
- [x] You implemented all the requirements✔️ 




## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Comment by musangamfure** [comment]
Thank you so much @algerina

**Pull request merged**